src/Runner/CallbackTaskRunner.php

```diff
@@ -39,7 +39,7 @@ public function run(TaskInterface $task): Output
 
             $task->setExecutionState(TaskInterface::SUCCEED);
 
-            return new Output($task, trim(!is_string($output) ? (string) $output : $output));
+            return new Output($task, trim(is_string($output) ? $output : (string) $output));
```

Comment:
```suggestion
            return new Output($task, trim((string) $output));
```
Not sure about this one